DockerfileLocal

```diff
@@ -0,0 +1,24 @@
+# Quickstart: Compose and Rails https://docs.docker.com/compose/rails/
+
+# Base image
+FROM ruby:2.5
+# Install dependencies
+RUN apt-get update -qq && apt-get install -y nodejs npm postgresql-client imagemagick cron
+WORKDIR /usr/src
+COPY Gemfile /usr/src/Gemfile
+COPY Gemfile.lock /usr/src/Gemfile.lock
+RUN bundle install
+RUN npm install -g bower
+COPY . /usr/src
+
+# Add a script to execute before starting main process
+COPY entrypoint.sh /usr/bin/
+RUN chmod +x /usr/bin/entrypoint.sh
+ENTRYPOINT ["entrypoint.sh"]
+EXPOSE 3001
+
+RUN useradd --create-home appuser
+USER appuser
+
+# Start the main process
+CMD ["rails", "server", "-b", "0.0.0.0"]
```

Comment:
This _should_ be alright now - I think this should default to Puma which is a perfectly cromulent webserver.  But I'd make sure to check the output and make sure it's not starting in WEBrick, bc that's insecure and painfully slow enough that I try to avoid it even on dev servers.

Reply:
It definitely comes up in Puma so this isn't an issue!